src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -369,6 +369,7 @@ public enum NuGetLogCode
         /// <summary>
         /// NuGetAudit enabled, but no vulnerability data found
         /// </summary>
+        /// <remarks>Was added in 6.7 client SDK, but then removed from use in 6.8</remarks>
         NU1905 = 1905,
```

Comment:
I'm undecided if I should delete this code, or leave it in. I still think the feature to warn when Audit doesn't run is useful, which customers can `WarningsAsErrors` in order to break their build if CI was unable to scan for vulnerable packages, in the cases where customers want to do that.

Reply:
I don't mind if it gets deleted. You can reintroduce the code if need at a later point. 

Your call though.